src/txHashBuilder.h

```diff
@@ -52,7 +53,8 @@ typedef enum {
 	TX_HASH_BUILDER_IN_MINT_TOP_LEVEL_DATA = 1010,
 	TX_HASH_BUILDER_IN_MINT_ASSET_GROUP = 1011,
 	TX_HASH_BUILDER_IN_MINT_TOKEN = 1012,
-	TX_HASH_BUILDER_FINISHED = 1100,
+	TX_HASH_BUILDER_IN_NETWORK_ID = 1100,
```

Comment:
This might need some adjustment when other tx body fields are added so that we don't have any clash. I'd leave the FINISHED value alone, though.

Reply:
Not sure how to make it certain it causes a merge conflict and demands attention then.

Reply:
I'm not sure either, but it seems to make sense to build all the other Alonzo features on top of this branch, and then we don't need to think about the conflict.

Reply:
Yeah, just occurred to me while finishing another alonso feature. Will rebase and rework that after the standup.